geode-core/src/main/java/org/apache/geode/internal/cache/backup/BackupTask.java

```diff
@@ -38,6 +42,7 @@
 class BackupTask {
   private static final Logger logger = LogService.getLogger();
 
+  private static final String CLUSTER_CONFIG_DISK_STORE_NAME = "cluster_config";
```

Comment:
Just a minor comment. Maybe to change the access modifier of "InternalConfigurationPersistenceService.CLUSTER_CONFIG_DISK_STORE_NAME" to public and use it instead of this one to avoid duplicated code. 

No further comments from my side, everything else looks good. Good work!

Reply:
Thanks. Updated.